From fb1a6ab205fd63a46669a463931af473e2ac0c87 Mon Sep 17 00:00:00 2001
From: Matthew Flatt <mflatt@racket-lang.org>
Date: Sat, 21 Aug 2021 15:29:59 -0600
Subject: [PATCH] setup/variant: recognize starter executables

Related to #3969

(cherry picked from commit 1629887071fe3cc8fe8af0a7aa0d3912509cb058)
---
 racket/collects/setup/variant.rkt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/racket/collects/setup/variant.rkt b/racket/collects/setup/variant.rkt
index 81da6f5701..b23131e481 100644
--- a/racket/collects/setup/variant.rkt
+++ b/racket/collects/setup/variant.rkt
@@ -25,7 +25,7 @@
         (and (file-exists? f)
              (with-input-from-file f
                (lambda ()
-                 (define m (regexp-match #rx#"bINARy tYPe:..(.)"
+                 (define m (regexp-match #rx#"bINARy tYPe:e?..(.)"
                                          (current-input-port)))
                  (cond
                    [(not m) '3m]
-- 
2.30.2

